config/initializers/paper_trail.rb

```diff
@@ -0,0 +1 @@
+PaperTrail.config.track_associations = false
```

Comment:
Please add the new line char. I have actually had issues when it was missing!
Thanks